### PR TITLE
[MO] Remove Apache MxNet models support in openvino-dev

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** Due to drop of Python 3.8, MxNet support is removed. MxNet is archived project and distributed only up to Python 3.8

**Ticket:** 153365

### CODE REVIEW

**Review by popovaan — APPROVED** [review]

**Review by mryzhov — APPROVED** [review]

**Review by andrei-kochin — CHANGES_REQUESTED** [review]
Waiting for the PxT approval

**Comment by rkazants** [comment]
> Waiting for the PxT approval

@andrei-kochin, @moslex, please discuss approval for this PR merge today.

Thanks,
Roman

**Review by andrei-kochin — APPROVED** [review]
Approved

**Pull request merged**
